pages/linux/equery.md

```diff
@@ -0,0 +1,23 @@
+# equery
+
+> View information about portage packages.
+
+- List all installed packages:
+
+`equery list '*'`
+
+- Search for installed packages:
+
+`equery list -po {{package_name}}`
+
+- List all packages depending on a package:
+
+`equery depends {{package_name}}`
+
+- List package dependencies:
```

Comment:
For consistency, I'd suggest something like "List all packages that a given package depends on" (and in the above example, "List all packages that depend on a given package". This makes the symmetry clearer and is more explicit.